src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1559,6 +1561,13 @@ def list_consumption_locations():
     return [{'name': x.name.lower().replace(" ", "")} for x in regions]
 
 
+def list_locations(sku, linux_workers_enabled=None):
+    client = web_client_factory()
+    full_sku = _get_sku_name(sku)
+    regions = client.list_geo_regions(full_sku, linux_workers_enabled)
+    return [{'name': x.name.lower().replace(" ", "")} for x in regions]
```

Comment:
I suggest the following, so that we would not remove all other properties coming up in future.
```python
    for x in regions:
        x.name = x.name.lower().replace(' ', '')
```